seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/fake/source/FakeSourceStream.java

```diff
@@ -67,12 +68,14 @@ public String getPluginName() {
     }
 
     private static final String[] NAME_ARRAY = new String[]{"Gary", "Ricky Huo", "Kid Xiong"};
+    private static final int AGE_LIMIT = 100;
 
     @Override
     public void run(SourceContext<Row> ctx) throws Exception {
+        Random random = new Random();
         while (running) {
             int randomNum = (int) (1 + Math.random() * NAME_ARRAY.length);
-            Row row = Row.of(NAME_ARRAY[randomNum - 1], System.currentTimeMillis());
+            Row row = Row.of(NAME_ARRAY[randomNum - 1], (long) random.nextInt(AGE_LIMIT));
```

Comment:
Use int here is ok, you don't need to convert to long. You can change the type in line 51